lang/src/main/java/mumbler/truffle/node/special/LambdaNode.java

```diff
@@ -12,17 +13,17 @@
 public abstract class LambdaNode extends MumblerNode {
     public abstract MumblerFunction getFunction();
 
+    @CompilerDirectives.CompilationFinal
```

Comment:
We usually import the @CompilationFinal directly and put it on the same line as the variable.

Reply:
Ok